pkgs/tools/networking/sendme/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sendme";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "n0-computer";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-UaAiHGeSqy4kHO9CZX3kYeECZDo45web6yMbBRVnlhQ=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
```

Comment:
greetings !

unfortunately hydra/ofbord doesn't allow import from derivation. you can read more here https://nixos.org/manual/nix/unstable/language/import-from-derivation

meanwhile, if Cargo.lock doesn't contain any dependencies fetched via git you can/should use `cargoHash` otherwise you will need to vendor/add Cargo.lock file into nixpkgs repository

 more info https://nixos.org/manual/nixpkgs/stable/#compiling-rust-applications-with-cargo

Reply:
hello @kirillrdy I forgot about this, thanks for the quick review!

There's an upstream crate (`sendme -> iroh -> watchable`) that makes using `cargo vendor` with the `cargoHash` method not buildable. A new version of `watchable` is fixed, I have a PR waiting in `iroh` now, and it can then eventually be fixed in `sendme`, but for now I've just commited the Cargo.lock file (can be removed in the future when this is all resolved)

Reply:
indeed I've tried and I got

```
error: couldn't read /build/sendme-0.3.0-vendor.tar.gz/watchable/src/.crate-docs.md: No such file or directory (os error 2)
 --> /build/sendme-0.3.0-vendor.tar.gz/watchable/src/lib.rs:1:10
```

maybe mention somewhere around lock file to check during next update to see if lock file can be removed

Reply:
add an inline TODO comment